content/english/anti-vegan/humans-are-superior.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "Humans Are Superior"
+date: 2019-10-12T07:28:15+13:00
+name: tery-hardwicke
+contributors:
+    - 
+        name: tery-hardwicke
+        role: author
+    - 
+        name: sam-martin
+        role: editor
+draft: false
+description: "How to respond to the argument: We are entitled to use animals as a resource because humans are superior to nonhuman animals"
+weight: 5
+---
+# Argument
+
+> ...but, humans are superior...  
+>   
+> We are entitled to use animals as a resource because humans are superior to nonhuman animals
+
+# Rebuttal
+
+## Quick Responses
+
+> Superiority supposes an arbitrary line that falsely identifies one group as deservedly superior to another.   
+
+and  
+ 
+> Yes, humans are unique and distinct from other animals in our capacity for complex problem solving. However, pick any species and you will likely find an attribute that also makes them unique and distinct. We should celebrate difference and not deny moral consideration to those who are different.
+ 
+or 
+  
+> No unique set of abilities or characteristics, in itself, provides a criterion for ‘superiority’.  For humans to claim that humans are superior because we have specific unique attributes is just another form of prejudice. 
+ 
+or
+  
+> Even *if* it is the case we are uniquely superior, morality requires we care for those more vulnerable than ourselves; not that we treat them as a resource.
```

Comment:
Sam-Martin 15 hours ago Member
Can we bump this to the top and consider stressing if rather than it is the case?

 @DrTeo
DrTeo 14 hours ago Author Member
Stressed if, didn't bump. Rather not. It's the logically weakest... (even if it's the one more likely to be used.)

 @Sam-Martin
Sam-Martin now Member
Still a reason to bump it though! Let's make these easy to find for people :)

(Copying back in as it got marked by github as outdated so hard to find).

Reply:
Yeah, struggling a bit today with the 'outdated' ... can get a little hard to follow.   I've sort of bumped it (combined aliens and this is single response